api/editor/api-editor/src/main/java/ca/bc/gov/tno/keycloak/SecurityConfig.java

```diff
@@ -33,11 +32,6 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
     auth.authenticationProvider(keycloakAuthenticationProvider);
   }
 
-  @Bean
```

Comment:
Had to place in separate file due to upgrading dependency.  Spring is piece of junk.